app/javascript/src/translation/en.json

```diff
@@ -12,7 +12,13 @@
   },
   "contact": {
     "delete": {
-      "success": "Contact has been deleted successfully!"
+      "success": "Contact deleted successfully."
+    },
+    "create": {
+      "success": "Contact added successfully."
+    },
+    "update": {
+      "success": "Contact updated successfully"
```

Comment:
There is a lot of redundancy in these keys. Can you take a closer look into these key-value pairs and get back to me on how we can make this more DRY?

Reply:
I guess we could restructure it in the following manner by passing in Contact/Note as an entity
```
{
  "success": {
    "delete": "{{Entity}} deleted successfully",
    "create": "{{Entity}} added successfully",
    "update": "{{Entity}} updated successfully",
  }
}
```